index.js

```diff
@@ -1,7 +1,23 @@
 var fs = require('fs');
 var bootstrap = fs.readFileSync(__dirname + '/output.js', 'utf8');
+var loaderUtils = require('loader-utils');
+var wasmDCE = require('wasm-dce')
 
 module.exports = function(buffer) {
+
+  var params = {};
+
+  if (this.resourceQuery !== "") {
+    params = loaderUtils.parseQuery(this.resourceQuery);
```

Comment:
Looks like we don't pass through `resourceQuery` in our specs, that's why the tests are failing. The tests aren't _integration_ per se(webpack itself isn't involved).

Reply:
Ok got it, i'll fix this later.